docassemble/ALToolbox/misc.py

```diff
@@ -138,7 +140,9 @@ def number_to_letter(n: Optional[int]) -> str:
     return string
 
 
-def collapse_template(template, classname=None):
+def collapse_template(
+    template, classname=None, expand_icon="caret-down", collapse_icon="caret-right"
```

Comment:
nit: I'd use `expanded_icon` and `collapsed_icon`, so it's clear that those are the icons being used in those states. Right now it's vague to me if expand should be the icon to click to expand it or the icon that shows when expanded.

Reply:
I considered that, but it was more confusing that way :) expand_icon is the one you click to expand it. It makes more sense to have the verbs match in my opinion.

Reply:
>  expand_icon is the one you click to expand it

True, but the default here is `caret-down`, which is what's shown when it's already expanded before this PR. So you might be confused too ;) should probably fix that.

Even if I don't agree with the naming, if you added an explanation to the doc string I'd be fine.

Reply:
Ah, I didn't mean to reverse it. I'll check on that.

Reply:
I changed the adverbs to "closed" and "open" to match the CSS and your suggestion--also good catch on the accidental reversing of the default!